### ISSUE TITLE
SSL_connect error on some https sites (not all)

### ISSUE THREAD

**Issue by dur3x** [Ysurac/openmptcprouter#146]
## Expected Behavior

root@OpenMPTCProuter:~# curl -v -k https://93.184.220.70
> GET / HTTP/1.1
> Host: 93.184.220.70
> User-Agent: curl/7.60.0
> Accept: */*
> 
< HTTP/1.1 404 Not Found
< Content-Type: text/html
< Date: Sun, 02 Sep 2018 13:11:20 GMT
< Server: ECS (lcy/1D68)
< Content-Length: 345
< 
<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
         "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
        <head>
                <title>404 - Not Found</title>
        </head>
        <body>
                <h1>404 - Not Found</h1>
        </body>
</html>

## Actual Behavior

bash-3.2$ curl -v -k https://93.184.220.70
* Rebuilt URL to: https://93.184.220.70/
*   Trying 93.184.220.70...
* TCP_NODELAY set
* Connected to 93.184.220.70 (93.184.220.70) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to 93.184.220.70:443 
* stopped the pause stream!
* Closing connection 0
curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to 93.184.220.70:443 

## Steps to Reproduce the Problem

Just note that I put all my traffic to the glorytun(tcp) but btw I think if it works directly on the router itself it should be the same on client.

  1. on the openmptcprouter: curl -v -k https://93.184.220.70 and see it works
  2. on a client do the same thing: curl -v -k https://93.184.220.70 and see it doesn't work

Note that this problem is not present on all https websites

## Specifications

  - OpenMPTCProuter version: (Last is not a version) v0.34.1
  - OpenMPTCProuter VPS version: (Last is not a version) OpenMPCTProuter VPS 0.46
  - OpenMPTCProuter platform: (RPI2/RPI3/x86/x86_64) x86_64

**Comment by Ysurac** [comment]
It's working for me on a client with default configuration.
What do you mean by putting all traffic using glorytun TCP ? You disable ShadowSocks ?

**Comment by dur3x** [comment]
Yes my goal was to put all the traffic through the glorytun tcp instead of shadow socks. I know it's not the best (tcp over tcp) but currently in my setup it's the most working one. In fact if I use the default configuration with shadowsocks then I have instabilities on my setup (btw it's another problem).
Perhaps there is a better way that I can try to do to that.

```
config ss_redir 'hi'
        option server 'sss0'
        option local_port '1100'
        option mode 'tcp_and_udp'
        option timeout '600'
        option fast_open '1'
        option verbose '0'
        option reuse_port '1'
        option mptcp '1'
        option ipv6_first '1'
        option local_address '0.0.0.0'
        option disabled 'true'

config ss_redir 'hi2'
        option server 'sss0'
        option local_address '0.0.0.0'
        option local_port '1100'
        option mode 'tcp_and_udp'
        option timeout '600'
        option fast_open '1'
        option verbose '0'
        option reuse_port '1'
        option mptcp '1'
        option ipv6_first '1'
        option disabled 'true'

config ss_rules 'ss_rules'
        option disabled '0'
        option redir_tcp 'hi'
        option src_default 'forward'
        option dst_default 'forward'
        option local_default 'forward'
        list dst_ips_forward '8.8.8.8'

config server 'sss0'
        option server_port '65101'
        option method 'chacha20'
        option server '37.187.101.205'
        option key 'Y7dioYAqKRibVexyraw/FsIv2EehEUtrWJGgc1qHFLg='
        option disabled '0'

config ss_tunnel 'dns'
        option disabled '1'
        option mode 'tcp_and_udp'
        option server 'sss0'
        option local_port '5353'
        option tunnel_address '8.8.8.8:53'

config ss_local 'tracker'
        option server 'sss0'
        option local_address '127.0.0.1'
        option local_port '1111'
        option mode 'tcp_and_udp'
        option timeout '600'
        option fast_open '1'
        option reuse_port '1'
        option mptcp '1'
        option verbose '0'
        option disabled 'true'

```

**Comment by Ysurac** [comment]
Fixed in OpenMPTCProuter VPS script 0.48. It's a MTU problem on gt-tun0 link.

**Comment by dur3x** [comment]
Thanks you are a boss ! It works perfectly now.
